### ISSUE TITLE
[GUIDE] Issues after upgrading to Njord Menu from menu.sh and advancemenu.sh

### ISSUE THREAD

**Issue by Nimdy** [Nimdy/Dedicated_Valheim_Server_Script#275]
Some folks are reporting issues after upgrading to the new menu.  This is more than likely because of the way Valheim was installed in the past.

After completing over 100 upgrade tests, I had zero issues.  

So if you are having issues after the upgrade, do not worry, you did not lose anything.

After the upgrade, we have the script create a backup of your file system and world files (whatever.db and whatever.fwl)

The .db and .fwl files are the only two files you should ever care about because recovering or reinstalling on another server only requires these two files.

If your system did not update correctly, please do the following.

1. Launch the Njord Menu and press option 3 -Install Valheim Server or Add another Valheim Instance

2. Complete the Valheim install process as normal.

3. Set your Public Name

4. Set your World Name (Name this your old world name)
--- You can find your old World Name by typing
`ls -lisa /home/steam/.config/unity3d/IronGate/Valheim/worlds/`

5. Set your Port information

6. Set your Public Listing

7. Set your Valheim Server Password for Access

8. Let it finishing installing

9. Stop your Valheim Service by typing
`systemctl start valheimserver_YOURWORLDNAMEHERE.service`

10. Copy over your .db and .fwl into the newly created directory
`cp /home/steam/.config/unity3d/IronGate/Valheim/worlds/YOURWORLDNAMEHERE.db /home/steam/.config/unity3d/IronGate/Valheim/YOURWORLDNAMEHERE/worlds/YOURWORLDNAMEHERE.db `
`cp /home/steam/.config/unity3d/IronGate/Valheim/worlds/YOURWORLDNAMEHERE.fwl /home/steam/.config/unity3d/IronGate/Valheim/YOURWORLDNAMEHERE/worlds/YOURWORLDNAMEHERE.fwl `

11. Make sure steam has the correct permissions
`chown -Rf steam:steam /home/steam/`

12. Start your Valheim Server
systemctl start valheimserver_YOURWORLDNAMEHERE.service

Everything should come up just fine after this.

Basically, what happen was during the upgrade the folder structure did not match the original installation configurations that were suppose to be setup.  Maybe this is because of the hosting provider or maybe something else. 

This upgrade works 100% if Njord Menu users are in baseline.

If you have any issues after words, please join our Discord and we will help you out.

Happy Gaming!
